internal/gitea/team.go

```diff
@@ -0,0 +1,149 @@
+package gitea
+
+import (
+	"fmt"
+	"log"
+	"slices"
+	"strings"
+
+	"code.gitea.io/sdk/gitea"
+	"github.com/jnsgruk/releasegen/internal/repos"
+)
+
+const giteaPerPage = 10
```

Comment:
```suggestion
const perPage = 10
```

Or maybe `reposPerPage` or something more specific?